src/index.ts

```diff
@@ -246,9 +252,19 @@ export class DebugProxy extends EventEmitter implements DebugProxyInterface {
     const possiblyHitBreakpointList = await Promise.all(possiblyHitPromiseList);
     possiblyHitBreakpointList.forEach((breakpoint: Breakpoint) => {
       if (breakpoint.isFinalState) {
-        assert(this.breakpointInfoMap.has(breakpoint.id));
-        this.breakpointInfoMap.get(breakpoint.id)![hit] = true;
-        hitAny = true;
+        const breakpointInfo = this.breakpointInfoMap.get(breakpoint.id);
+        if (!breakpointInfo) {
+          throw new Error(
+              `The breakpoint with ID ${breakpoint.id} ` +
```

Comment:
Done.

Reply:
Almost every function in this file could throw, because every function in wrapper.ts could throw. Would it still be useful to document this in the interface for each function? Or should I just make a top-level comment that almost every method could throw?

Reply:
That's really up to you.  However, the more detailed you can be the better.  I think this case is more important as this isn't something I would expect.

Reply:
OK, I've added annotations for errors that aren't obviously errors, unlike the ones in wrapper.ts. For this specific line, the throw is because I'm paranoid, and I've added a comment there.